src/Api/ExternalConfig.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @Author: Erik Slovak <erik.slovak@technologystudio.sk>
+ * @Date: 2024-04-12T14:23:07+02:00
+ * @Copyright: Technology Studio
+**/
+
+import type * as yup from 'yup'
+import { realpathSync } from 'fs'
+import untildify from 'untildify'
+import update from 'immutability-helper'
+
+import { type ConfigShape } from '../Model/Types'
+
+import {
+  getConfig,
+  setConfigName,
+} from './GetRc'
+
+const CONFIG_PATH = '~/.txo/config.json'
+
+setConfigName(CONFIG_PATH)
+
+export const getConfigMap = (configSchema: yup.ObjectSchema<ConfigShape>): ConfigShape => {
```

Comment:
profile fallbacks are not implement/extracted here

Reply:
This returns the whole map where keys are profile names. Should I add the functionality for specific profiles?

Reply:
yes, that was the puprose to implement support of profiles.